rmf_task/src/rmf_task/BackupFileManager.cpp

```diff
@@ -114,6 +125,11 @@ class BackupFileManager::Robot::Implementation
   std::filesystem::path robot_directory;
```

Comment:
To be consistent, let's also make `robot_directory` `const` like the rest of the string and path fields.

Reply:
did the same for other paths in this commit 905d1811d1a6e8b94c26cdc33c64eb984b48c8bb